File: 3516. Find Closest Person/Solution.py
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
      # if distance between x and z is lower then return 1 else 2.
      # we will use abs to calculate absolute difference, 
        if abs(z-x) < abs(z-y):
            return 1
        elif abs(z-x) > abs(z-y):
            return 2
        else:
            return 0

File: 3516. Find Closest Person/Solution.cpp
class Solution {
public:
    int findClosest(int x, int y, int z) {

        // Checking and comparing the distance between the two pair of persons
        // If x is closer to z return 1
        if(abs(z-x)<abs(z-y)) return 1;
        // If y is closer to z return 2
        else if(abs(z-x)>abs(z-y)) return 2;
        // If both x and y arrives at same time, return 0
        else return 0;
        
    }
};
